android/KMEA/app/src/main/java/com/tavultesoft/kmea/JSONParser.java

```diff
@@ -43,6 +45,13 @@ public <T extends Object> T getJSONObjectFromReader(BufferedReader reader, Class
       }
       jsonStr = strBuilder.toString();
       if (type == JSONObject.class) {
+        String EMPTY_JSONARRAY_FORMATSTR = "^\\[\\](\\n)?$";
```

Comment:
```suggestion
        String EMPTY_JSONARRAY_FORMATSTR = "^\\[\\](\\n)*$";
```

Allow for more than one blank end line?